.github/workflows/pytest-gpu.yml

```diff
@@ -56,24 +56,30 @@ jobs:
         -resourceGroupNameRegion $RESOURCE_GROUP_REGION
         -serverName $SERVER_NAME
         -vmImage $IMAGE
-        -adminLogin ${{ secrets.ADMIN_LOGIN }}
-        -adminPassword ${{ secrets.ADMIN_PASSWORD }}
+        -adminLogin ${{ secrets.CI_GPU_VM_ADMIN_LOGIN }}
+        -adminPassword ${{ secrets.CI_GPU_VM_ADMIN_PASSWORD }}
 
     - name: set host
-      run: echo ::set-output name=action_host::$(az vm show -d -g ${{ secrets.RESOURCE_GROUP }} -n gpu_ci_vm_${{ steps.extract_branch.outputs.branch }} --query publicIps -o tsv)
+      run: echo ::set-output name=action_host::$(az vm show -d -g ${{ secrets.CI_GPU_RESOURCE_GROUP }} -n gpu_ci_vm_${{ steps.extract_branch.outputs.branch }} --query publicIps -o tsv)
       id: host
 
     - name: execute tests on VM
       uses: fifsky/ssh-action@master
       with:
         command: |
           #! /bin/bash
-          # Ensure clang and cuda libraries are visible
-          export PATH=~/programs/install/bin:$PATH
-          export LD_LIBRARY_PATH=~/programs/install/lib:$LD_LIBRARY_PATH
+          # Ensure clang, PGI and cuda libraries are visible
+          export PATH=/usr/local/cuda-10.1/bin:$PATH
+          export LD_LIBRARY_PATH=/usr/local/cuda-10.1/lib64:$LD_LIBRARY_PATH
+          export PATH=~/install/bin:$PATH
+          export LD_LIBRARY_PATH=~/install/lib:$LD_LIBRARY_PATH
+          export PGI=/opt/pgi
+          export PATH=/opt/pgi/linux86-64/19.10/bin:$PATH
+          export MANPATH=$MANPATH:/opt/pgi/linux86-64/19.10/man
+          export LM_LICENSE_FILE=$LM_LICENSE_FILE:/opt/pgi/license.dat-COMMUNITY-19.10
```

Comment:
This really need to go somwhere on the VM, this is a massive extra here.

Reply:
Maybe we should add all the paths in .bashrc of the image?

Reply:
Yeah. Iirc putting the exports wasn't working with the action previously - some weird interaction with the action. But I'll play around and put them somewhere suitable.

Reply:
I did a tried before on exporting an image from a VM and it was retained in .bashrc. I will try a bit later again.

Reply:
@FabioLuporini I don't agree. These are compiler and VM specif path that are not workflow related

Reply:
I meant the _action_ I think?

Reply:
But these are still VM/node/.... environment variables/ setups, not devito or pytest parameters.  What if you switch linux distribution, or want to test it on osx? you gonna hve 200 lines of exports/env vars that should not be here.

Reply:
Is there any verdict here? Should I try any other approach? Ideas?

Reply:
I agree with @mloubout